lib/methods/export.js

```diff
@@ -290,16 +332,16 @@ module.exports = self => {
           });
         }
       }
+    },
 
-      function relatedTypesIncludes(name) {
-        if ([ '@apostrophecms/any-page-type', '@apostrophecms/page' ].includes(name)) {
-          return relatedTypes.some(type => {
-            const module = self.apos.modules[type];
-            return self.apos.instanceOf(module, '@apostrophecms/page-type');
-          });
-        }
-        return relatedTypes.includes(name);
+    relatedTypesIncludes(name, relatedTypes) {
+      if ([ '@apostrophecms/any-page-type', '@apostrophecms/page' ].includes(name)) {
+        return relatedTypes.some(type => {
+          const module = self.apos.modules[type];
+          return self.apos.instanceOf(module, '@apostrophecms/page-type');
+        });
```

Comment:
@boutell I can move this function in a utils file if you think it makes no sense to attach it to self (which I would agree with I think).